src/mmrelay/cli_utils.py

```
@@ -691,34 +701,62 @@ async def logout_matrix_bot(password: str) -> bool:
                 )
-                print("⚠️  Logout response unclear, proceeding with local cleanup.")
+            else:
+                if hasattr(logout_response, "transport_response"):
+                    logger.info("Successfully logged out from Matrix server.")
+                    print("✅ Successfully logged out from Matrix server.")
+                else:
+                    logger.warning(
+                        "Logout response unclear, proceeding with local cleanup."
+                    )
+                    print("⚠️  Logout response unclear, proceeding with local cleanup.")
         except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)